tests/ui/type-alias-impl-trait/type-alias-impl-trait-fns.rs

```diff
@@ -1,5 +1,3 @@
-// check-pass
-
```

Comment:
This PR "reopens" https://github.com/rust-lang/rust/issues/61863

the test doesn't really add too much new test coverage compared with other tests we already have.